README.md

```diff
@@ -1,4 +1,107 @@
+- Need to check to make sure text above any header at the top of a document is handled appropriately.
+- Test: What happens if the user hit save multiple times rapidly before Send To Deck completes when it occurs on save?
+- Don't send if no changes to file. Important for Send All command so it doesn't check every file every time needlessly. Probably need to add last-sent meta timestamp to compare to file mod time.
+- Make sure added features are working with "Send to Own Deck"
+- Auto-hide/collapse anki metadata in editor so you don't have to look at it.
+
+## Known Issues
+
+- What should SendAll do if multiple workspaces? 
+- At the moment, Nested header strategy ignores clozes, % sign and tags. I don't use any of this, so I am in no rush to add it.
+- Nested Header strategy ignores original author's use of title to send to own deck.
+- Card updates from within Anki are overwritten. Is there a solution...? There doesn't seem to be any way to know if the card was updated in Markdown or Anki last and which should take priority.
+- MarkdownFile uses node os.EOL for line breaks. Should probably be consistent with code, since a user could I think, use a non-os linebreak system within code.
+- Send to Deck is calls save. On didsave calls send. So likely activated twice if the file is already being tracked.
+
+
+
+## Nested Headers
+
+This is a strategy to create cards from markdown. The way it works is the header #, ##, ###, etc delimit cards. Each block of html between headers becomes the back of a card.
+The front of the cards is the tree of headers to the card. This allows you to create cards like so:
+
+axillary-neuropraxia.md:
+
+      # Axillary Neuropraxia
+
+      see [[Neuropaxia]]
+
+      ## Etiology
+
+      - Shoulder dislocation
+      - Masses
+
+      ## Findings
+
+      ### MRI findings
+
+      Atrophy of teres minor muscle, inferior deltoid
+
+      ![](2021-01-15-10-39-54.png) ![](2021-01-15-10-41-15.png)
+
+      ## Associated Lesion
+
+      [[GAGL]] from shoulder dislocation
+
+Text appearance of created cards (In reality they have the HTML styling from the markdown parser):
+
+    Axillary Neuropraxia
+    ---
+    see [[Neuropaxia]]
+
+    
+    Axillary Neuropraxia
+    Etiology
+    ---
+    * Shoulder dislocation 
+    * Masses
+
+    
+    Axillary Neuropraxia
+    Findings
+    MRI findings
+    ---
+    Atrophy of teres minor muscle, inferior deltoid 
+    img img***
+
+    
+    Axillary Neuropraxia
+    Associated Lesion
+    ---
+    [GAGL] from shoulder dislocation
+
+*** Media & html is sent to and rendered in Anki appropriately like the original plugin.
+
+I recommend you update the card style sheet within Anki to adjust the headers, in particular to make h1 smaller. I just added the line  `h1 { font-size: 20px }` to mine.
+
+
+# Original Readme below:
+
+
```

Comment:
Can we move this text into the Pull Request thread and keep the readme the same for now?

Reply:
I don't mean to send my readme. Github just automatically send sent everything.